frontend/javascripts/oxalis/controller/combinations/segmentation_plane_controller.js

```diff
@@ -100,6 +100,6 @@ export function isosurfaceLeftClick(pos: Point2, plane: OrthoView, event: MouseE
   cellId = segmentation.cube.getMappedDataValue(position, getRequestLogZoomStep(Store.getState()));
 
   if (cellId > 0) {
-    Store.dispatch(changeActiveIsosurfaceCellAction(cellId, position));
+    Store.dispatch(changeActiveIsosurfaceCellAction(cellId, position, true));
```

Comment:
`true` is or `shouldReload`, right? Wouldn't this be unnecessary for precomputed meshes? Or is that case caught in the isosurface saga (I couldn't find that, though)?

Reply:
The function doesn't even seem to be used, so maybe I can just remove it?

Reply:
Yes, good point :+1: